kaitaistruct.py

```diff
@@ -15,6 +15,18 @@
 #
 __version__ = '0.8'
 
+from RecursionSafe import RecursionSafe
+
+recSafe=RecursionSafe()
```

Comment:
does flake8 not complain about that symbol name?

Reply:
recSafe will be fixed to `rec_safe`. I don't use flake.